app/locale/en.coffee

```diff
@@ -1495,11 +1503,12 @@
     end_date: "end date:"
     num_enrollments_needed: "Number of licenses needed:"
     get_enrollments_blurb: " We'll help you build a solution that meets the needs of your class, school or district."
-    enroll_request_sent_blurb1: "Thanks! Your request has been sent."
-    enroll_request_sent_blurb2: "Our classroom success team will be in touch shortly to help you find the best solution for your students' needs!"
-    enroll_request_sent_blurb3: "Please reach out to <a href='mailto:schools@codecombat.com'>schools@codecombat.com</a> if you have additional questions at this time."
+    how_to_apply_licenses_blurb_1: "When a teacher assigns a course to a student for the first time, we’ll automatically apply a license. Use the bulk-assign dropdown in your classroom to assign a course to selected students:"
+    how_to_apply_licenses_blurb_2: "Can I still apply a license without assigning a course?"
+    how_to_apply_licenses_blurb_3: "Yes - go to the License Status tab in your classroom and click \"Apply License\" to any student who does not have an active license."
```

Comment:
Not sure the best way to use em-dashes in translated stuff, but I've been using `&mdash;`. I have my own personal pet peeve of misusing hyphens tho.